ConfigSpace/configuration_space.py

```diff
@@ -233,7 +233,7 @@ def add_conditions(self, conditions: List[ConditionComponent]) -> List[Condition
                 edges.extend(
                     [(dlc.parent, dlc.child) for dlc in dlcs])
                 values.extend([dlc.value for dlc in dlcs])
-                conditions_to_add.extend(dlcs)
+                conditions_to_add.extend([condition] * len(dlcs))
```

Comment:
Could you please explain this statement?

Reply:
Okay, this seems to fix your statement about adding Conjunctions. Could you please provide a failing example for this?